### PR TITLE
big warning on top of the old authenticate api requests guide

### PR DESCRIPTION
**Pull request by freben**
I thought maybe we leave it in place for a short bit, especially since it mentions the need to remove those changes. Could also consider removing entirely, of course

### CODE REVIEW

File: contrib/docs/tutorials/authenticate-api-requests.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # Authenticate API requests
 
+> [!CAUTION]
```
Comment (benjdlambert) [inline contrib/docs/tutorials/authenticate-api-requests.md:3]: Probably should use `::note` annotations as they look better. You can do warnings as well https://github.com/backstage/backstage/pull/24473

Comment (freben) [reply contrib/docs/tutorials/authenticate-api-requests.md:3]: is that the same here in contrib rather than the docs folder?

Comment (benjdlambert) [reply contrib/docs/tutorials/authenticate-api-requests.md:3]: Ah I didn't catch that this was not docs, but yeah, you can do it in github markdown too.

Comment (benjdlambert) [reply contrib/docs/tutorials/authenticate-api-requests.md:3]: Actually scrap that - what you have is good.

**Review by benjdlambert — APPROVED** [review]

**Pull request merged**